public_src/components/sidebar/validation-browser.component.ts

```diff
@@ -68,12 +90,52 @@ export class ValidationBrowserComponent {
    * @returns {void}
    */
   public startNameCorrection(): void {
-    this.appActions.actSetErrorCorrectionMode('missing name tag');
+    if (this.errorCorrectionMode.refSuggestions) {
+    this.appActions.actSetErrorCorrectionMode(
+      {
+        nameSuggestions: {
+          found          : true,
+          startCorrection: true,
+        },
+        refSuggestions : this.errorCorrectionMode.refSuggestions,
+      }); }
+      else {
+      this.appActions.actSetErrorCorrectionMode(
+        {
+          nameSuggestions: {
+            found          : true,
+            startCorrection: true,
+          },
+          refSuggestions : this.errorCorrectionMode.refSuggestions,
+        });
+    }
     this.errorHighlightSrv.missingTagError('name');
   }
 
-  public startRefCorrection(): any {
-    this.appActions.actSetErrorCorrectionMode('missing ref tag');
+  /***
```

Comment:
the doc. comments started with `/**` everywhere in the project. Since the last PR +- you started writing `/***`